ManageTeamsIdentityMobileAndDesktop/src/main/java/com/communication/quickstart/App.java

```diff
@@ -0,0 +1,62 @@
+package com.communication.quickstart;
+
+import com.azure.communication.identity.CommunicationIdentityClient;
+import com.azure.communication.identity.CommunicationIdentityClientBuilder;
+import com.azure.communication.identity.models.GetTokenForTeamsUserOptions;
+import com.azure.core.credential.AccessToken;
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import com.microsoft.aad.msal4j.InteractiveRequestParameters;
+import com.microsoft.aad.msal4j.PublicClientApplication;
+
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Set;
+
+public class App {
+
+    public static void main( String[] args ) throws Exception {
+        System.out.println("Azure Communication Services - Communication access token Quickstart");
+        // Quickstart code goes here
+
+        // You need to provide your Azure AD client ID and tenant ID
+        String appId = "<contoso_application_id>";
+        String tenantId = "<contoso_tenant_id>";
+        String authority = "https://login.microsoftonline.com/" + tenantId;
+
+        // Create an instance of PublicClientApplication
+        PublicClientApplication pca = PublicClientApplication.builder(appId)
+                .authority(authority)
+                .build();
+
+        String redirectUri = "http://localhost";
+        Set<String> scope = new HashSet<String>();
+        scope.add("https://auth.msft.communication.azure.com/Teams.ManageCalls");
+        scope.add("https://auth.msft.communication.azure.com/Teams.ManageChats");
+
+        // Create an instance of InteractiveRequestParameters for acquiring the AAD token and object ID of a Teams user
+        InteractiveRequestParameters parameters = InteractiveRequestParameters
+                .builder(new URI(redirectUri))
+                .scopes(scope)
+                .build();
+
+        // Retrieve the AAD token and object ID of a Teams user
+        IAuthenticationResult result = pca.acquireToken(parameters).get();
+        String teamsUserAadToken = result.accessToken();
+        String[] accountIds = result.account().homeAccountId().split("\\.");
+        String userObjectId = accountIds[0];
+        System.out.println("Teams token: " + teamsUserAadToken);
+
+        // You can find your connection string from your resource in the Azure portal
+        String connectionString = "<connection_string>";
```

Comment:
would it be possible to have all the configs in the top? (easier to find all values that need to be replaced)

Reply:
I can move it to the top, but is it really neccessary? as we mentioned in the README and will be easy find it in 60lines code

Reply:
not necessary, just a suggestion. I leave that to your judgement :)

Reply:
I prefer to declare variables exactly when they are first used and also in other quickstarts are done in the same way, so I am closing this thread :)